frontend/src/pages/PlaceOrder.jsx

```
@@ -44,71 +43,67 @@ const PlaceOrder = () => {
+    }
+    return orderItems;
+  };
+  const placeCODOrder = async (orderData, token) => {
+    const res = await axios.post(backendUrl + "/api/order/place", orderData, {
+      headers: { token },
+    });
+    return res.data;
+  };
+
+  const placeStripeOrder = async (orderData, token) => {
+    const res = await axios.post(backendUrl + "/api/order/stripe", orderData, {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)